nav2_bringup/launch/nav2_bringup_2nd_launch.py

```diff
@@ -25,8 +25,8 @@ def generate_launch_description():
             parameters=[params_file]),
 
         launch_ros.actions.Node(
-            package='dwb_controller',
-            node_executable='dwb_controller',
+            package='nav2_follow_path_controller',
```

Comment:
Agree. I like 'nav2_control_action', 'nav2_action_controller', or 'nav2_controller_action'

Reply:
What about just `nav2_local_controller` or `nav2_controller`? Throwing the word action and server in there sounds a bit awkward to me.

Reply:
I think that’s fine too, I might encourage renaming dwb to dwb_local_planner_plugin then to sufficiently distinguish between the controlLER and the control-Action

Reply:
From the diagram on #943, I understand this package is `local_planner` and the node executable is `local_planner_executor`

Reply:
well, `nav2_local_planner` you mean? I can get behind that. I can't find a specific definition for "executor" but my feeling* (which could be totally, totally wrong) is that executor is not really what this describes. It's the action server "stuff" that wraps the algorithmic controlling "stuff", so I feel like `local_planning_server` for the executable would be in line with that

Reply:
The nomenclature we have traditionally used is `local planner` rather than `controller`. I don't have a strong opinion either way, but if we call it `nav2_controller` we should update any reference to `local_planner` as now controller, minux DWB